src/test/java/org/apache/sysml/test/gpu/UnaryOpTests.java

```diff
@@ -82,6 +82,11 @@ public void testAtan() throws Exception {
 	}
 
 	@Test
+	public void testSVD() throws Exception {
```

Comment:
I am pretty sure this will break. Your `svd` function requires multiple outputs, yet the `testSimpleUnaryOpMatrixOutput` doesn't provide multiple outputs.